views/js/runner/plugins/controls/progressbar/progressbar.js

```diff
@@ -32,6 +32,11 @@ define([
     'use strict';
 
     /**
+     * Display 'item x of y' when true, 'item x' otherwise
+     */
+    var progressShowTotal;
```

Comment:
Why is this variable global to the module? You should rather make it local to the plugin instance, like `progressIndicator` and `progressScope`.